configs/platforms/debian-10-amd64.rb

```diff
@@ -0,0 +1,10 @@
+platform "debian-10-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "buster"
+
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot cmake"
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
```

Comment:
you aren't installing anything here, is this intended?

Reply:
this command will be used to install extra packages that are required using `build_requires`